### ISSUE TITLE
Differentiating between a complex key phrase of multiple words vs. proximity search for multiple words

### ISSUE THREAD

**Issue by porta-antiporta** [18F/fec-eregs#188]
TL:DR;  Need to differentiate in our search criteria how to treat multiple words as a single search term.

Our current search mechanism allows the user to enter multiple keywords into the field.  When that is done, the search function uses proximity search (find documents where those two words are mentioned within ~20 words) to find search results.

However, there are times when a user wants to search for a key phrase (more than 1 word) such as `electioneering communications` or `travel reimbursement`; they are looking for documents that match that phrase exactly. Or at the very least within much closer proximity than 20 words, such as `reimbursement for travel`.

Our current UI is unable to differentiate when two words should be treated as a single key phrase (ex: `travel reimbursement`) or two separate key words (ex `filings briefs`).

The purpose of this story is to identify what the best design/UX pattern is to allow a user to make that distinction.

-----------------
@fec-issue-bot commented on [Tue Jul 19 2016](https://github.com/18F/FEC/issues/402)

## What were you trying to do and how can we improve it?
Search for regulations addressing electioneering communications

## General feedback?
wanted to be able to see only electioneering communication results, not results for &#34;communication&#34; (i.e. ex parte communications, public communication . . . )

## Details
* URL: https://beta.fec.gov/data/legal/regulations/?search=electioneering+communication
* User Agent: Mozilla/5.0 (Windows NT 6.1; rv:47.0) Gecko/20100101 Firefox/47.0

---

@porta-antiporta commented on [Tue Jul 19 2016](https://github.com/18F/FEC/issues/402#issuecomment-233767888)

Thanks for the feedback!  Agreed, this is a similar finding to https://github.com/18F/fec-eregs/issues/160 where we need to understand how to differentiate between complex terms and simple terms during search.

Moving this issue to fec-eregs repo to track.

**Comment by edmullen** [comment]
I believe that elasticsearch already has conventions for this. I believe it was wrapping terms in quotes identified it as an exact match. But that may be all backend. We might need to account for that in the UI. 

In general, I think we should adhere to google's conventions. https://support.google.com/websearch/answer/2466433?hl=en

I designed a design for assisted search guidance here: https://github.com/18F/fec-eregs/issues/94
